### PR TITLE
useCacheUntil option in install command 

### PR DESCRIPTION
**Pull request by sambathl**
useCacheUntil option to install command, since it is 1 minute by hard coded which isn't good

### CODE REVIEW

**Comment by pascalduez** [comment]
I'm not familiar with the CLI codebase, but this seems like a good addition.

**Comment by pascalduez** [comment]
I'm not familiar with the CLI codebase, but this seems like a good addition.

File: cli/src/lib/npm/npmLibDefs.js [hunk 1]
```diff
@@ -329,12 +329,15 @@ async function _npmExists(pkgName: string): Promise<Function> {
   return got(pkgUrl, {method: 'HEAD'});
 }
 
+export const CACHE_REPO_EXPIRY = 1000 * 60;
```
Comment (villesau) [inline cli/src/lib/npm/npmLibDefs.js]: this is already defined in 
https://github.com/flow-typed/flow-typed/blob/master/cli/src/lib/cacheRepoUtils.js#L11

Maybe export that and use it instead?

File: cli/src/commands/install.js [hunk 2]
```diff
@@ -147,6 +154,7 @@ export async function run(args: Args) {
     overwrite: Boolean(args.overwrite),
     skip: Boolean(args.skip),
     ignoreDeps: ignoreDeps,
+    useCacheUntil: Number(args.useCacheUntil) | CACHE_REPO_EXPIRY,
```
Comment (villesau) [inline cli/src/commands/install.js]: Did you forgot one pipe `||`? `|` is bitwise OR.

File: cli/src/lib/npm/npmLibDefs.js [hunk 3]
```diff
@@ -329,12 +329,15 @@ async function _npmExists(pkgName: string): Promise<Function> {
...
+
 export async function findNpmLibDef(
   pkgName: string,
   pkgVersion: string,
   flowVersion: FlowVersion,
+  useCacheUtil: number = CACHE_REPO_EXPIRY,
```
Comment (villesau) [inline cli/src/lib/npm/npmLibDefs.js]: I think this is now third place where default `CACHE_REPO_EXPIRY` is set.

Other places being:
- https://github.com/flow-typed/flow-typed/pull/3305/files#diff-8fda159841136e265107671260f39d80R157
- https://github.com/flow-typed/flow-typed/blob/master/cli/src/lib/cacheRepoUtils.js#L79

would https://github.com/flow-typed/flow-typed/blob/master/cli/src/lib/cacheRepoUtils.js#L79 be enough?

**Comment by sambathl** [comment]
@villesau okay to merge ? :smile_cat:

File: cli/src/lib/npm/npmLibDefs.js [hunk 4]
```diff
@@ -333,8 +334,9 @@ export async function findNpmLibDef(
...
 ): Promise<null | NpmLibDef> {
-  const libDefs = await getCacheNpmLibDefs();
+  const libDefs = await getCacheNpmLibDefs(useCacheUtil);
```
Comment (villesau) [inline cli/src/lib/npm/npmLibDefs.js]: should this be `useCacheUntil` instead of `useCacheUtil`? :)

**Comment by sambathl** [comment]
@villesau completed review remarks :smile:

**Comment by villesau** [comment]
Thanks @GAntoine!

**Pull request merged**